spec/app_templates/config/initializers/clearance.rb

```diff
@@ -0,0 +1,2 @@
+Clearance.configure do |config|
```

Comment:
Unused block argument - `config`. You can omit the argument if you don't care about it.